src/services/API/setToken.js

```diff
@@ -0,0 +1,17 @@
+import { createAccessTokenEndpoint } from "./endpoints";
+
+export function setToken(apiKey, unique_id) {
+  return fetch(createAccessTokenEndpoint, {
```

Comment:
Also, if you look at the other APIs, we use axios for API calling.

Reply:
done